pyproject.toml

```diff
@@ -97,19 +92,12 @@ filterwarnings = [
   "ignore:.*The --rsyncdir command line argument and rsyncdirs config variable are deprecated:DeprecationWarning",
   # Remove once https://github.com/protocolbuffers/protobuf/issues/12186 is fixed
   "ignore:.*custom tp_new.*in Python 3.14:DeprecationWarning",
-  # Remove once support for python 3.7 is dropped
-  # This warning only appears when using python 3.7
-  "ignore:.*Using or importing the ABCs from.*collections:DeprecationWarning",
-  # Remove once support for grpcio-gcp is deprecated
-  # See https://github.com/googleapis/python-api-core/blob/42e8b6e6f426cab749b34906529e8aaf3f133d75/google/api_core/grpc_helpers.py#L39-L45
-  "ignore:.*Support for grpcio-gcp is deprecated:DeprecationWarning",
-  "ignore: The `compression` argument is ignored for grpc_gcp.secure_channel creation:DeprecationWarning",
-  "ignore:The `attempt_direct_path` argument is ignored for grpc_gcp.secure_channel creation:DeprecationWarning",
+
+
   # Remove once the minimum supported version of googleapis-common-protos is 1.62.0
   "ignore:.*pkg_resources.declare_namespace:DeprecationWarning",
   "ignore:.*pkg_resources is deprecated as an API:DeprecationWarning",
   # Remove once https://github.com/grpc/grpc/issues/35086 is fixed (and version newer than 1.60.0 is published)
   "ignore:There is no current event loop:DeprecationWarning",
-  # Remove after support for Python 3.7 is dropped
-  "ignore:After January 1, 2024, new releases of this library will drop support for Python 3.7:DeprecationWarning",
```

Comment:
maybe we should keep in a generic version of this warning, since we expect to start seeing it for other versions at some point in the future